Tests/BootCamp.Chapter.Tests/Input/Transactions.csv

```diff
@@ -1,8 +1,9 @@
 ﻿Shop, City, Street, Item, DateTime, Price
 Kwiki Mart, Springfield, HoloPapap 2, Sausage, 2020-01-05T20:00:00Z, "€8"
+Kwiki Mart, Springfield, HoloPapap 2, Apple, 2020-01-05T20:00:00Z, "€3"
```

Comment:
Added to ensure that double grouping is performed to calculate the average earnings per hour of day. As the data was given, a single grouping `transactions.GroupBy(x => x.Time.Hour)` could pass the test and be an incomplete implementation.

Reply:
The tests are not thorough, to that I agree. Thanks!